htdocs/admin/company.php

```diff
@@ -75,21 +75,29 @@
 		activateModulesRequiredByCountry($mysoc->country_code);
 	}
 
-	$db->begin();
```

Comment:
Why removing the db->begin of transaction ?

Reply:
My mistake, sorry!